runtime/jcl/common/jcldefine.c

```diff
@@ -95,14 +95,15 @@ defineClassCommon(JNIEnv *env, jobject classLoaderObject,
 	/* Allocate and initialize a UTF8 copy of the Unicode class-name */
 	if (NULL != className) {
 		j9object_t classNameObject = J9_JNI_UNWRAP_REFERENCE(className);
-		utf8Name = (U_8*)vmFuncs->copyStringToUTF8WithMemAlloc(currentThread, classNameObject, J9_STR_NULL_TERMINATE_RESULT | J9_STR_XLAT, "", 0, utf8NameStackBuffer, J9VM_PACKAGE_NAME_BUFFER_LENGTH, &utf8Length);
+		UDATA stringFlags = (validateName ? J9_STR_NULL_TERMINATE_RESULT : (J9_STR_NULL_TERMINATE_RESULT | J9_STR_XLAT));
```

Comment:
This would read better as an assignment of the constant bit and an if to OR in the second bit.

Reply:
Updated the code against the suggestion above.